README.md

```diff
@@ -71,6 +71,9 @@ When you initialize your project via `Initialize Settings` the plugin will add t
             // To use non-standard ssh specify the path here
             "ssh_binary": "/usr/local/bin/ssh",
 
+            // You may also want to specify rsync executable path, for instance, if using MSYS under Windows
```

Comment:
I think the comment is too specific. Specifying the path is equally useful on any platform. Even on Mac where new OS versions strip the PATH passed to GUI applications like Sublime.
Just don't mention MSYS or Windows please.